trunk/NDHMS/Data_Rec/module_namelist.F

```diff
@@ -803,7 +807,9 @@ subroutine rt_nlst_check(nlst)
          if (.not. fileExists) call hydro_stop('hydro.namelist ERROR: reservoir_parameter_file not found.')
       endif
    end if
-
+   if( (nlst%lake_shape_option .lt. 0 ) .or. (nlst%lake_shape_option .gt. 3) ) then
```

Comment:
It looks like there is an option -1 in the code below.  Is this not active?

Reply:
Initially it was -1 then I changed that to 0. So it should be options 0-3 everywhere!